packages/components/src/components/UtilityCard/UtilityCard.tsx

```diff
@@ -77,7 +77,7 @@ export const UtilityCard: React.FC<UtilityCardProps> = ({
         padding={{ _: "d4", md: "d6" }}
         w={{ _: "56px", md: "56px" }}
       >
-        <Box.img alt="" aria-hidden h="24px" src={imageSrc} w="24px" />
+        <Text.p>{emoji}</Text.p>
```

Comment:
I think we don't even need a paragraph here. Having only the emoji string should be enough.